syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -13,6 +13,23 @@
 from syft.exceptions import PureTorchTensorFoundError
 
 
+def _get_maximum_precision():
```

Comment:
I'm not 100% sure about this solution though it is a way of allowing different setups. If you can think on a better alternative, happy to discuss it

Reply:
Yes maybe this can be a bit more flexible. What I experienced is that dealing with integers > 2**62-1 is not fun with precision tensors, so 62 could be the total limit which sum the space allocated to the fractional_precision *and* the space allocated to the integer part. I think we should find a way to take this into account.
Another point to consider: we shouldn't mix precision in bits and decimal precision: from the user interface, people usually keep the `base` at 10 so the precision they ask for (3 by default) means 3 digits after the `.` which roughly corresponds to a 10 bits precision